github/github_test.go

```diff
@@ -111,9 +111,14 @@ func testFormValues(t *testing.T, r *http.Request, values values) {
 }
 
 func testHeader(t *testing.T, r *http.Request, header string, want string) {
-	if got := r.Header.Get(header); got != want {
-		t.Errorf("Header.Get(%q) returned %q, want %q", header, got, want)
+	relevantHeaders := r.Header[header]
```

Comment:
Sorry, @davidlopezre - this change should be reverted, as it is not correct.  See below.